programs/test/udp_proxy.c

```diff
@@ -773,7 +773,7 @@ int main(int argc, char *argv[])
 
     struct timeval *tm_ptr = NULL;
 
-    int nb_fds;
+    SOCKET nb_fds;
```

Comment:
`SOCKET` only works on Windows. Maybe `net_sockets.h` should export a type `mbedtls_net_socket_handle_t`.

Reply:
The whole idea of this PR is to make code portable.
SOCKET in non-Windows OS becomes int.

Reply:
Now the code does not need `SOCKET` macro.